pkg/common/fileutils/fileutils.go

```diff
@@ -124,6 +125,29 @@ func GetAllFilesInDirectory(path string) []string {
 	return retVal
 }
 
+func GetAllFilesWithSameNameInDirectory(path string, fname string) []string {
```

Comment:
Where is this function going to be used. Be careful where this func is going to get called from since there could be thounsands of files in dir. 
also maybe there is a better os utility package available that can do exact/wildcard match of whatever file you are looking for : for e.g. ls myfile*

Reply:
You should be able to use `filepath.Walk` to do what a bit cleaner; I'm not sure if there's some other function that does exactly what you want

Reply:
Addressed in the latest commit.